django/core/mail/message.py

```diff
@@ -354,7 +354,9 @@ def _create_attachments(self, msg):
             encoding = self.encoding or settings.DEFAULT_CHARSET
             body_msg = msg
             msg = SafeMIMEMultipart(_subtype=self.mixed_subtype, encoding=encoding)
-            if self.body:
+            # Attach the body_msg if self.body isn't empty or
+            # body_msg is a list of sub-EmailMessage objects
```

Comment:
Is `EmailMessage` correct? When running tests, I see `body_msg` is something like `SafeMIMEText` or `SafeMIMEMultipart`. Please use a period at the end of the sentence.